demo/spec/home.spec.js

```diff
@@ -1,8 +1,14 @@
+/* eslint-env jest */
+
 import React from 'react';
 import renderer from 'react-test-renderer';
-import { Home } from '../src/Home';
+import ConnectedHome from '../src/home';
+
+const { WrappedComponent: Home } = ConnectedHome;
```

Comment:
Nice! Accessing the static property `WrappedComponent` instead of explicitly exporting the unconnected component is an interesting (not in the bad way) approach.